src/core/qgsmaprenderer.h

```diff
@@ -335,6 +335,14 @@ class CORE_EXPORT QgsMapRenderer : public QObject
      */
     bool splitLayersExtent( QgsMapLayer* layer, QgsRectangle& extent, QgsRectangle& r2 );
 
+    /** Set a feature filter provider to filter the features
+     * @param ffp the feature filter provider
+     * @note not available in Python bindings
```

Comment:
Just a general question, what the general criterion to add or not in the python interface?